includes/class-wc-stripe.php

```diff
@@ -668,37 +639,6 @@ protected function enable_upe( $settings ) {
 		return $settings;
 	}
 
-	protected function disable_upe( $settings ) {
```

Comment:
Removed the method entirely, as it is a private method and there's no use in disabling UPE anymore.